profiler/lib.profiler/external/binaries-list

```diff
@@ -16,3 +16,4 @@
 # under the License.
 45225DCFC94A9782419E95C70957822A0C44612D profiler-external-binaries-8.2.zip
 B1C1B951F79716EFFAB9BEC96A611844B038E2E2 profiler-external-binaries-ASF-5273a10f9471d5cf60ed894dc26ec0e59e71cc24.zip
+AD1F88D2B35874756A192ACD033A17AAD8C6535C https://dist.apache.org/repos/dist/dev/netbeans/native/netbeans-profiler/1-r24aefa9/profiler-external-binaries-1-24aefa9.zip profiler-external-binaries-1-24aefa9.zip
```

Comment:
Release is done, this can now be updated to:

```suggestion
AD1F88D2B35874756A192ACD033A17AAD8C6535C https://archive.apache.org/dist/netbeans/native/netbeans-profiler/1-24aefa9/profiler-external-binaries-1-24aefa9.zip profiler-external-binaries-1-24aefa9.zip
```